sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1212,6 +1220,50 @@ def table_fn(element, fictional_characters):
   # [END model_bigqueryio_time_partitioning]
 
 
+def model_bigqueryio_xlang(
+    pipeline, write_project='', write_dataset='', write_table=''):
+  """Examples for cross-language BigQuery sources and sinks."""
+
+  table_spec = 'clouddataflow-readonly:samples.weather_stations'
```

Comment:
```suggestion
  table_spec = 'clouddataflow-readonly:samples.<non-existent-table>'
```
so everything else can stay the same, just put a table name that doesn't exist. can even use `str(uuid.uuid4())` in the name to make sure

Reply:
thank you, I added your suggestion. It seems it works! 👍